### PR TITLE
Support unquoted space separated value

### PR DESCRIPTION
**Pull request by ulyssessouza**
This adds support for unquoted space separated values

### CODE REVIEW

File: fixtures/quoted.env [hunk 1]
```diff
@@ -1,14 +1 @@
...
OPTION_E="1"
OPTION_F="2"
OPTION_G=""
OPTION_H="\n"
OPTION_I = "echo 'asd'"
OPTION_J = 'first line
second line
third line
and so on'
OPTION_Z = "last value"
```
Comment (ndeloof) [inline fixtures/quoted.env:1]: are you using a Windows computer now?

File: parser.go [hunk 2]
```diff
@@ -221,3 +221,9 @@ func isSpace(r rune) bool {
 	}
 	return false
 }
+
+
+// isNewLine reports whether the rune is a new line character
+func isNewLine(r rune) bool {
+	return r == '\n'
```
Comment (ndeloof) [inline parser.go:228]: should also detect `\r` so CR LF on windows will detect the first one, otherwise we will cut after CR

**Review by ndeloof — CHANGES_REQUESTED** [review]
Not sure ^M are intentional here ?
need support for `\r\n`

File: parser.go (hunk 2 above)
Comment (ulyssessouza) [reply parser.go:234]: Fixed by just trimming the end on any kind of space char when unquoted.

Comment (ndeloof) [reply parser.go:234]: ok, make sense

**Review by ndeloof — APPROVED** [review]

**Pull request merged**